root/etc/e-smith/templates/etc/httpd/conf.d/default-virtualhost.inc/50evebox

```diff
@@ -0,0 +1,24 @@
+{
+    use esmith::ConfigDB;
+    use esmith::util;
+    my $confdb = esmith::ConfigDB->open;
+    my $evebox = $confdb->get('evebox') or die "No evebox db entry found\n";
+    my $alias = $evebox->prop('alias') || "";
+    my $port = $evebox->prop('TCPPort') || "5636";
+
+    # initialize alias if needed
+    if ($alias eq "") {
+        $alias = esmith::util::genRandomHash();
+        $confdb->set_prop('evebox','alias',$alias);
+    }
+
+    $OUT = "
+RewriteEngine On
+RewriteCond %{HTTP_HOST} !^localhost [NC]
+RewriteCond %{REMOTE_ADDR} !^127\.0\.0\.1\$
+RewriteCond %{HTTPS} !=on
+RewriteRule ^/$alias(/.*)?\$  https://%{HTTP_HOST}/$alias\$1  [L,R=301]
```

Comment:
One question: I understand the `default-virtualhost.inc` file does not need to configure the ProxyPass because it already "inherits" the seting from the Server scope, right?

Reply:
Done

Reply:
Yes, exactly